### ISSUE TITLE
The training loss is so weird

### ISSUE THREAD

**Issue by kkkoaoa** [ashawkey/stable-dreamfusion#367]
### Description

<img width="1920" alt="b18ab1c10244d785ffd98caed7b9d40" src="https://github.com/user-attachments/assets/dc89d589-76d5-450f-8419-ec822730d3e1">

https://github.com/user-attachments/assets/1c1b31a7-8170-4e99-8abc-21677909809c

https://github.com/user-attachments/assets/9cdc9e36-25e5-45c1-9d5f-dda8f2e3acb1

### Steps to Reproduce

1.Training statements：python main.py --text "a hotdog" --workspace trial2 -O2
2.It's the first training session after the previous steps
3.Its training loss is particularly strange, either large in the tens of thousands or zero

### Expected Behavior

normal loss not ab normal loss

### Environment

pip install torch==2.1.2+cu118 torchvision==0.16.2+cu118 --extra-index-url https://download.pytorch.org/whl/cu118

**Comment by Lpigge** [comment]
issue65 have same problem？maybe you can check it and get something.

**Comment by hongyi-h** [comment]
> issue65 have same problem？maybe you can check it and get something.

I checked #65. However, this loss problem remains.
